packages/scripts/src/utils/utils.ts

```diff
@@ -1,7 +1,50 @@
 const hre = require("hardhat");
+import nodeFetch from "node-fetch";
 const { ethers } = hre;
 
 export const increaseEvmTime = async (time: number): Promise<void> => {
   await ethers.provider.send("evm_increaseTime", [time]);
   await ethers.provider.send("evm_mine", []);
 };
+
+export const takeSnapshot = async (): Promise<string> => {
+  const snapshot = await ethers.provider.send("evm_snapshot", []);
+  return snapshot;
+};
+
+export const revertToSnapshot = async (snapshotId: string): Promise<void> => {
+  await ethers.provider.send("evm_revert", [snapshotId]);
+};
+
+export const forkNetwork = (jsonRpcUrl: string, blockNumber?: string): Promise<void> => {
+  return hre.network.provider.request({
+    method: "hardhat_reset",
+    params: [
```

Comment:
is this the best formatting we can do here? I think it could be over fiewer lines like:
`params: [{ forking: { jsonRpcUrl, blockNumber } }]`